wavelength-plugins/tsumeta.js

```diff
@@ -209,17 +243,18 @@ exports.commands = {
 	},
 
 	tsumetahelp: [
-		`/tsumeta give [user] - Gives a user the TsuMeta Council Member status.`,
-		`/tsumeta take [user] - Removes a user's TsuMeta Council Member status.`,
-		`/tsumeta list - Shows the list of TsuMeta Council Members.`,
-		`/tsumeta alert [message] - Sends a message to all online users from the TsuMeta Council. Only for Desokoro.`,
-		`/tsumeta propose [what you modified], [change requested] - Proposes a change for the TsuMeta metagame. Must be in the TsuMeta Council to use.`,
-		`/tsumeta proposals [optional proposal ID] - Checks the specified proposal ID, if not specified generates a random one from the proposals index.`,
-		`/tsumeta deleteproposal [proposal] - Deletes the specified proposal. Reserved for Desokoro and xcmr.`,
-		`/tsumeta viewproposals - Shows the list of proposals.`,
-		`/tsumeta suspend [target] - Suspends a user from proposing/participating in the TsuMeta council. Only for Desokoro and xcmr.`,
-		`/tsumeta unsuspend [target] - Unsuspends a user from proposing/participating in the TsuMeta council. Only for Desokoro and xcmr.`,
-		`/tsumeta forums - Displays the official TsuMeta Website.`,
-		`/tsumeta help - Displays this help command.`,
+		`/tsumeta give [user] - Gives a user the TsuMeta Council Member status.
+		/tsumeta take [user] - Removes a user's TsuMeta Council Member status.
+		/tsumeta list - Shows the list of TsuMeta Council Members.
+		/tsumeta alert [message] - Sends a message to all online users from the TsuMeta Council. Only for Desokoro.
+		/tsumeta propose [what you modified], [change requested] - Proposes a change for the TsuMeta metagame. Must be in the TsuMeta Council to use this command.
+		/tsumeta edit [proposal], [description update] - Edits a proposal. Must be the submitter of the proposal, Desokoro, or xcmr.
+		/tsumeta proposals [optional proposal ID] - Checks the specified proposal ID, if not specified generates a random one from the proposals index.
+		/tsumeta deleteproposal [proposal] - Deletes the specified proposal. Reserved for Desokoro and xcmr.
+		/tsumeta viewproposals - Shows the list of proposals.
+		/tsumeta suspend [target] - Suspends a user from proposing/participating in the TsuMeta council. Only for Desokoro and xcmr.
+		/tsumeta unsuspend [target] - Unsuspends a user from proposing/participating in the TsuMeta council. Only for Desokoro and xcmr.
+		/tsumeta forums - Displays the official TsuMeta Website.
+		/tsumeta help - Displays this help command.`,
```

Comment:
Why is it all one giant string now?

Reply:
I'm pretty sure that the padding after each newline stays within the template string. If so, these should be separate.

Reply:
Honestly I thought it looked cleaner to read

Reply:
@HoeenCoder 1 giant template string with line breaks (like how it is) displays it with spacings between lines

Reply:
Just looks a hell of a lot easier to read code-wise.